### DISCUSSION TITLE
Easiest way to deploy local jekyll website with mm theme to github pages

### DISCUSSION THREAD

**Discussion by Davidyao217** [mmistakes/minimal-mistakes#3565] [Q&A]
I've been following along with mm's website's guide and the github official docs, but so far the build just fails everytime. Does anyone have any advice or a general approach they took?

**Comment by iBug** [comment]
From README:

> **Looking for an example?** Use the [Minimal Mistakes remote theme starter](https://github.com/mmistakes/mm-github-pages-starter/generate) for the quickest method of getting a GitHub Pages hosted site up and running. Generate a new repository from the starter, replace sample content with your own, and configure as needed.
